tools/report-converter/tests/unit/sparse_output_test_files/sample.out

```diff
@@ -0,0 +1,6 @@
+files/sample.c:4:1: warning: symbol 'machine_id' was not declared. Should it be static?
+./files/sample.h:3:40: warning: incorrect type in argument 2 (different address spaces)
```

Comment:
One file starts with `./` others are not. Why is that. It should be consistent.

Reply:
Hmm yes, I think we can remove this line from the sample.out since we are already testing the header files below the note line